internal/jennies/python/tools.go

```diff
@@ -0,0 +1,60 @@
+package python
+
+import (
+	"fmt"
+	"strings"
+)
+
+type raw string
+
+func formatValue(val any) string {
+	if val == nil {
+		return "None"
+	}
+
+	if rawVal, ok := val.(raw); ok {
+		return string(rawVal)
+	}
+
+	if asBool, ok := val.(bool); ok {
+		if asBool {
+			return "True"
+		}
+
+		return "False"
+	}
+
+	if list, ok := val.([]any); ok {
+		items := make([]string, 0, len(list))
+
+		for _, item := range list {
+			items = append(items, formatValue(item))
+		}
+
+		return fmt.Sprintf("[%s]", strings.Join(items, ", "))
+	}
+
+	return fmt.Sprintf("%#v", val)
+}
+
+func escapeFieldName(name string) string {
+	if isReservedPythonKeyword(name) {
+		return name + "_val"
```

Comment:
Does it changes the json result? I don't recall if Python uses the field names as json keys.

Reply:
I didn't reach this PR yet :)